roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -104,3 +104,16 @@ cifmw_cephadm_grafana_container_image: "{{ cifmw_cephadm_container_ns }}/ceph-gr
 cifmw_cephadm_node_exporter_container_image: "{{ cifmw_cephadm_prometheus_container_ns }}/node-exporter:v1.5.0"
 cifmw_cephadm_prometheus_container_image: "{{ cifmw_cephadm_prometheus_container_ns }}/prometheus:v2.43.0"
 cifmw_cephadm_ns: openstack
+cifmw_cephadm_cert_name: cert-ceph-rgw
+cifmw_cephadm_cert_dns_names:
```

Comment:
should this be a list populated by the edpm nodes hosts?
```
cifmw_cephadm_cert_dns_names:
 - compute-0.cluster.local
 - compute-1.cluster.local
 - compute-2.cluster.local
 ```
In theory this can also be generated/populated in a task, but good to have a default variable that we can eventually override. 
In theory when we include `rgw_cert.yaml` tasks we already computed the host list, and that will be available as a fact and used to compute `dns_names`.

Reply:
Yes, I think we don't need a separate variable for `cifmw_cephadm_cert_dns_names`. The latest version drops that in favor of using the established `_hosts` fact.